docs/source/release_notes/release-notes.rst

```diff
@@ -40,6 +40,10 @@ develop
     *    - Type
          - Change
 
+    *    - |changed|
+         - Default 'gc_grace_seconds' set by AtlasDB for Cassandra tables has been changed from four days to an hour.  This will allow Cassandra to start cleaning up swept data sooner after sweeping.
```

Comment:
Yea we need to provide simple instructions for how to modify current tables to the new value, or an automated way of doing it.